CONTRIBUTING.md

```diff
@@ -33,6 +33,7 @@ This `egghead-ui` repo does two things:
 - `yarn dev:library` to develop the library
 - `yarn verify` check linting and builds work
 - [localhost:2000](http://localhost:2000) to view the style guide
+- run `yarn build` to build updated components
```

Comment:
I would recommend removing this; it is much easier to just keep `yarn dev:library` running as mentioned in the 4th bullet point of "Workflow" - as this will watch the build so you don't have to manually build after each change. Make sense?